jpos/src/main/java/org/jpos/q2/Q2.java

```diff
@@ -810,22 +821,22 @@ private void deployBundle (File bundle, boolean encrypt)
         }
     }
     public void deployElement (Element e, String fileName, boolean encrypt, boolean isTransient)
-        throws ISOException, IOException, GeneralSecurityException
+        throws IOException, GeneralSecurityException
```

Comment:
Might be better to leave the ISOException here. It's an API method and subclasses might have over-ridden this method.